eng/testing/tests.mobile.targets

```diff
@@ -147,11 +147,19 @@
       <ExtraAssemblies Include="$(PublishDir)System.Xml.dll" />
       <ExtraAssemblies Include="$(PublishDir)WindowsBase.dll" />
     </ItemGroup>
+    <!-- these tests load assemblies from the file system -->
     <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.MetadataLoadContext.Tests'">
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Private.CoreLib.dll" />
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.MetadataLoadContext.Tests.dll" />
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)mscorlib.dll" />
     </ItemGroup>
+    <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.Metadata.Tests'">
+      <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.Metadata.Tests.dll" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.Tests'">
+      <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.Tests.dll" />
```

Comment:
is there a better way to track these dependencies? hardcoding looks wrong

Reply:
Or avoid defining this centrally and instead set a property in the project file that indicates this?

Reply:
yeah we need to find a better way.

Reply:
Can you please open an issue to discuss this?

Reply:
I think the project could add to an item and then we could transform that item to remap the value to be under `PublishDir`